package.json

```diff
@@ -37,32 +37,32 @@
     "test:e2e:firefox:grid": "mocha --firefox --grid",
     "test:e2e:opera": "mocha --opera",
     "test:e2e:safari": "mocha --safari",
-    "test:server:start": "http-server &",
+    "test:server:start": "http-server -p 8080 &",
```

Comment:
Waarom moet de poort gedefinieerd worden?

Reply:
Moeten niet, maar wel handig dat het dezelfde poort is als wat gedefinieerd wordt voor selenium. Als poort 8080 in gebruik is, zal die een foutmelding geven.